src/lib.rs

```diff
@@ -74,13 +75,17 @@ use nalgebra::convert;
 pub use nalgebra::Vector3;
 
 use hal::blocking::delay::DelayMs;
+use hal::blocking::i2c;
 use hal::blocking::spi;
 use hal::digital::OutputPin;
 use hal::spi::{Mode, Phase, Polarity};
 
 pub use conf::*;
 pub use types::*;
 
+#[doc(hidden)]
```

Comment:
why hide docs here? I understand why it might be ok for Releasable, but I don't quite get why hide docs for *Device. After all, users will have to use those types in explicit type signatures.

Reply:
Type signature compatibility didn't cross my mind! Thanks for bringing it up. Yup, these will all need to be exposed.